config/frontend/config.json

```diff
@@ -1,79 +1,125 @@
-   "multipleDownloadAction":"",
-   "multipleDownloadEnabled":false,
-   "oAuth2Endpoints":[],
-   "policiesEnabled":true,
-   "retrieveDestinations": [],
-   "riotBaseUrl":"",
-   "scienceSearchEnabled":true,
-   "scienceSearchUnitsEnabled":true,
-   "searchPublicDataEnabled":true,
-   "searchSamples":true,
-   "sftpHost":"",
-   "shareEnabled":false,
-   "shoppingCartEnabled":false,
-   "shoppingCartOnHeader":false,
-   "tableSciDataEnabled":true
-}
+{
+  "accessTokenPrefix": "Bearer ",
+  "addDatasetEnabled": true,
+  "archiveWorkflowEnabled": false,
+  "datasetReduceEnabled": true,
+  "editDatasetSampleEnabled": true,
+  "editMetadataEnabled": true,
+  "editPublishedData": false,
+  "editSampleEnabled": true,
+  "externalAuthEndpoint": "/auth/msad",
+  "facility": "SAMPLE_SITE",
+  "loginFacilityLabel": "SAMPLE_SITE",
+  "loginLdapLabel": "Ldap",
+  "loginLocalLabel": "Local",
+  "loginFacilityEnabled": true,
+  "loginLdapEnabled": true,
+  "loginLocalEnabled": true,
+  "fileColorEnabled": true,
+  "fileDownloadEnabled": true,
+  "gettingStarted": null,
+  "ingestManual": null,
+  "jobsEnabled": true,
+  "jsonMetadataEnabled": true,
+  "jupyterHubUrl": "",
+  "landingPage": "doi.invalid/detail/",
+  "lbBaseURL": "http://localhost:80",
+  "localColumns": [
+    {
+      "name": "select",
+      "order": 0,
+      "type": "standard",
+      "enabled": true
+    },
+    {
+      "name": "datasetName",
+      "order": 1,
+      "type": "standard",
+      "enabled": true
+    },
+    {
+      "name": "runNumber",
+      "order": 2,
+      "type": "standard",
+      "enabled": true
+    },
+    {
+      "name": "sourceFolder",
+      "order": 3,
+      "type": "standard",
+      "enabled": true
+    },
+    {
+      "name": "size",
+      "order": 4,
+      "type": "standard",
+      "enabled": true
+    },
+    {
+      "name": "creationTime",
+      "order": 5,
+      "type": "standard",
+      "enabled": true
+    },
+    {
+      "name": "type",
+      "order": 6,
+      "type": "standard",
+      "enabled": true
+    },
+    {
+      "name": "image",
+      "order": 7,
+      "type": "standard",
+      "enabled": true
+    },
+    {
+      "name": "metadata",
+      "order": 8,
+      "type": "standard",
+      "enabled": true
+    },
+    {
+      "name": "proposalId",
+      "order": 9,
+      "type": "standard",
+      "enabled": true
+    },
+    {
+      "name": "ownerGroup",
+      "order": 10,
+      "type": "standard",
+      "enabled": false
+    },
+    {
+      "name": "dataStatus",
+      "order": 11,
+      "type": "standard",
+      "enabled": false
+    }
+  ],
+  "logbookEnabled": false,
+  "loginFormEnabled": true,
+  "maxDirectDownloadSize": 5000000000,
+  "metadataPreviewEnabled": true,
+  "metadataStructure": "",
+  "multipleDownloadAction": "https://downloadserver.invalid/zip",
+  "multipleDownloadEnabled": true,
+  "oAuth2Endpoints": [
+    { "authURL": "api/v3/auth/oidc", "displayText": "SSO OIDC" }

```

Comment:
should this be empty? otherwise it will display the OIDC form which will break as the be is not configured

Reply:
The problem is that you can't add comments in json. The idea was to show more or less all available features and how to configure them.  But I agree, having things non-functional may also be a bad idea.

Reply:
maybe one could put one config.example.json with all possible values, maybe alongside a README with a brief explanation and a config.json with the default values for the containers (and do the same for all configs)? I saw those .example.json files in other scicat components.  If this is fine, I would do in another PR

Reply:
Yes, that would be an option.